### DISCUSSION TITLE
disable error msg upon silent

### DISCUSSION THREAD

**Discussion by HolyBugx** [projectdiscovery/naabu#574] [Q&A]
### Naabu version:

2.1.1

### Current Behavior:

Naabu version 2.1.1 errors out if the given host does not resolve.

```
❯ echo asdasd.iquweyjbxzc.com | naabu -silent -p 443

[FTL] Could not run enumeration: no valid ipv4 or ipv6 targets were found
```

### Expected Behavior:

Old Naabu versions do not have this problem. Here is the example test on version 2.0.6:

```
❯ echo asdasd.iquweyjbxzc.com | naabu -silent -p 443

❯
```

Best Regards.
HolyBugx

**Comment by ehsandeep** [comment]
@HolyBugx it's unclear what's the issue here. Do you mean error should not be displayed when `-silent` option is used or something else?

**Comment by HolyBugx** [comment]
Hi @ehsandeep,

True. If the error is intentional, it would be helpful to remove it when `-silent` is used.

Thanks.

**Reply by ehsandeep** [reply]
@HolyBugx error gets printed only when single input is passed, not in the case of multiple inputs.

disabling the error completely may cause more issues for troubleshooting on the user side.

**Reply by HolyBugx** [reply]
HI @ehsandeep,

True. It also happens if a file only has one entry. Is there a way to fix this and prevent the error? 

The error causes some problems when used in distributed automation.

Kind regards,
@HolyBugx

**Comment by olearycrew** [comment]
We are not going to disable errors in this case given the issues that could cause with troubleshooting.
